Makefile.dep

```diff
@@ -58,6 +58,20 @@ ifneq (,$(filter ng_ipv6_hdr,$(USEMODULE)))
   USEMODULE += ng_pktbuf
 endif
 
+ifneq (,$(filter ng_ipv6_router,$(USEMODULE)))
+  USEMODULE += ng_ipv6
+  USEMODULE += ng_fib
```

Comment:
Why is this dependency? (I understand the conceptional dependency, but I couldn't find the implementational dependency.)